src/scss/components/topnav/mixins/_topNavControl.scss

```diff
@@ -30,22 +30,20 @@
 @mixin __topNavControl--disabled($pseudo-state: null) {
   cursor: not-allowed;
 
-  hx-icon, p {
-    color: $gray-600;
+  hx-icon,
+  p {
+    color: var(--hxTopNav-disabled-icon-color, $gray-600);
   }
   @if $pseudo-state == hover {
-    background-color: $gray-950;
+    background-color: var(--hxTopNav-disabled-hover-color, $gray-950);
```

Comment:
Please write 
background-color: var(--hxTopNav-disabled-hover-backgroundColor, $gray-950);

Reply:
changed to --hxTopNav-hxTopNavControl-disabled-hover-bgcolor